index.html

```diff
@@ -3652,12 +3652,6 @@ <h3 id="applying">
         notation are defined in [[!ECMASCRIPT]].
       </p>
       <p>
-        When instructed to <dfn>Trim</dfn>(<var>x</var>), a user agent MUST
-        behave as if [[!ECMASCRIPT]]'s <a data-cite=
-        "!ECMASCRIPT#sec-string.prototype.trim">String.prototype.trim()</a>
```

Comment:
I wonder if @marcoscaceres needs to update the ECMASCRIPT reference is one of the respec tools instead.